pkg/kudoctl/cmd/init.go

```diff
@@ -111,26 +121,35 @@ func (i *initCmd) run() error {
 		return nil
 	}
 
+	// initialize client
+	if err := i.initialize(); err != nil {
+		return fmt.Errorf("error initializing: %s", err)
+	}
+	fmt.Fprintf(i.out, "$KUDO_HOME has been configured at %s.\n", Settings.Home)
+
 	// initialize server
-	if i.client == nil {
-		client, err := kube.GetKubeClient(viper.GetString("kubeconfig"))
-		if err != nil {
-			return fmt.Errorf("could not get kubernetes client: %s", err)
+	if !i.clientOnly {
```

Comment:
I spent non-trivial time trying to figure out what is `i` :) can we find a better name?

Reply:
I agree here, it's small but it's common to shorten in the context of methods, but use the longer name when assigning it to a variable.